examples/jenkins-master/README.md

```diff
@@ -0,0 +1,92 @@
+Jenkins Master->Slave Example
+=============================
+This sample walks through the process of starting up an OpenShift cluster and deploying a Jenkins Pod in it.
+It also leverage the [kubernetes-plugin](https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin) to manage
+the Jenkins slaves that run as on-demand Pods.
+
+The sample job that this Jenkins example contains will execute an unit test for
+a simple Ruby application. The unit test will be executed in a pod managed by
+the kubernetes plugin.
+
+Because the Jenkins Slave pod needs an Docker image that connects to the Jenkins
+when it starts, this sample also provide a template that allows you to convert
+any OpenShift S2I image to a valid Jenkins Slave image.
+
+This template is not required, if you already have Docker image that has the
+script that launch the [slave
+agent](https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly)
+as an entrypoint. In that case, you have to create the image stream for this
+image and use that image stream instead when creating the Jenkins master.
+
+Also note that the kubernetes plugin is already included in the official Jenkins
+image, so you don't have to use the template in this example. The template in
+this example will customize the Jenkins to install Git plugin and the sample
+Jenkins job used in this example.
```

Comment:
I think you need to better clarify the purpose of the template here.  

"The Jenkins image includes the Kubernetes plugin, so it can manage slave pods by default.  This tutorial includes a template which further customizes the Jenkins image by adding the Git plugin and define the sample job used by the tutorial."